spec/fixtures/stubs/cda/cases_body.json

```diff
@@ -0,0 +1,339 @@
+{
```

Comment:
should we call it with something that reminds the api call?
I called the response for the hearing_results this way `cda/internal_v2_hearing_results_response.json`